src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -565,7 +595,7 @@ public async Task UpdateTypeColumnWithArgument(string type, object value)
         /// Utility function to do special comparisons for some of the extended types
         /// if json compare doesn't suffice
         /// </summary>
-        private static void PerformTestEqualsForExtendedTypes(string type, string expected, string actual)
+        public static void PerformTestEqualsForExtendedTypes(string type, string expected, string actual)
```

Comment:
Do we need `public` or is `protected` sufficient?

Reply:
yes, protected can be used.